12/umbraco-commerce/reference/storefront-api/endpoints/order.md

```diff
@@ -0,0 +1,59 @@
+# Order Endpoints
+
+The Order endpoints are where you will manage your carts/orders and perform cart management functionality such as adding items to the cart, or setting up billing and shipping details.

```

Comment:
In order to keep it to 25 words, this would be my suggestion:
```suggestion
The Order endpoints are where you will manage your carts/orders and perform cart management functionality. Some examples are adding items to the cart, or setting up billing and shipping details.
```

Reply:
Updated 👍